logging/logging.go

```diff
@@ -139,16 +141,22 @@ func SetLogFile(filename string) {
 		return
 	}
 
-	fp, err := os.OpenFile(filename, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
-	if err != nil {
-		loggingFp = nil
+	w := &lumberjack.Logger{
+                        Filename:   filename,
+                        MaxSize:    100, // megabytes
+                        MaxBackups: 5,
+                        MaxAge:     5, // days
+                        Compress:   true,
+                    }
+	if w == nil {
 		fmt.Fprintf(os.Stderr, "multus logging: cannot open %s", filename)
```

Comment:
@shahar-klein why would `w` be nil here?

Reply:
Thanks for the comment - fixed.